Makefile

```diff
@@ -1,3 +1,7 @@
+VERSION=$$(git describe --tags --always)
+
+LDFLAGS=-ldflags=all="-X main.version=${VERSION}"
```

Comment:
I'm not sure if `all=` is needed here. Also I prefer `LDFLAGS` only defined "the flags" and not the whole command. Something like:

```
LDFLAGS="-X main.version ....."
 ...
build:
    go build -ldflags="$(LDFLAGS)" ...
```

Also to prevent collision from existing environment variable I prefer this to be called `GOLDFLAGS`.

Reply:
Good call. I will make this update as well.